inputfiles/addedTypes.jsonc

```
@@ -273,6 +273,16 @@
     },
     "enums": {
         "enum": {
+            "FontDisplay": {
+              "name": "FontDisplay",
+              "value": [
```

Looks good! One small nit, could you add a spec link for this, since it's a tad difficult to find where this is from:

(Yes it's in MDN, but the spec is always the source of trust)

```suggestion
              // https://w3c.github.io/csswg-drafts/css-fonts-4/#font-display-desc
              "value": [
```